dedibox-network/rpn/how-to/use-rpn-san-esxi.mdx

```diff
@@ -0,0 +1,182 @@
+---
+meta:
+  title: How to mount a RPN SAN volume in VMware ESXi
+  description: This page explains how to mount a RPN SAN volume in VMware ESXi
+content:
+  h1: How to mount a RPN SAN volume in VMware ESXi
+  paragraph: This page explains how to mount a RPN SAN volume in VMware ESXi
+dates:
+  validation: 2021-11-10
+  posted: 2021-11-10
+---
+
+<Message type="requirement">
+
+* You have an account and are logged into the [Dedibox Console](https://console.online.net)
+* You have a Dedibox dedicated server
+* You have a RPN SAN 
+
+</Message>
+
+<Message type="important">
+
+This documentation covers VMWare ESXi version 5.0/5.1/5.5 and 6.0.
+
+</Message>
+
+## How to configure VMWare ESXi to use RPN
+
+### Enable SSH on your ESXi
+
+1. Start the vSphere client or log in using the web interface. 
+
+2. Click the Configuration tab (1), then click **Security Profile** (2) in the software panel.
+
+  <Lightbox src="scaleway_esxi-san-en-01.png" /> 
```

Comment:
this image isn't showing for me in a yarn develop...but I guess this might be a general yarn image problem, because I can see the image file is there in assets.

Reply:
guess this a yarn develop issue, I have also some issue with the images